survey-runner-queue/ebs.tf

```diff
@@ -1,5 +1,5 @@
 module "ebs_bckup" {
-  source = "github.com/kgorskowski/ebs_bckup?ref=b08b842cac687af3dda4ccfc342bc4c2c0c692e5"
+  source = "github.com/ONSdigital/ebs_bckup?ref=7f49b01c9766ffdc427afe18fe3cd212ca077ee1"
```

Comment:
This ref needs to be updated but you already know that. just putting this review here to block it